src/CFamily/Rules/CFamilyRulesConfigProvider.cs

```diff
@@ -24,11 +24,9 @@
 using SonarLint.VisualStudio.Core;
 using SonarLint.VisualStudio.Core.Binding;
 using SonarLint.VisualStudio.Core.CFamily;
+using SonarLint.VisualStudio.Integration;
 
-// Note: currently this class needs to be in the the Integration assembly because it
-//   references IHost and other internal interfaces.
```

Comment:
We did some refactorings in April/May 2020 which mean "host" is no longer required.

I tried to move the class to Core, but it uses MEF so I moved it to CFamily instead.